Sourcery/Parsing/Parser.swift

```diff
@@ -364,16 +348,52 @@ final class Parser {
         }
 
         for (_, type) in unique {
+            //find actual variables types
             for variable in type.variables {
-                if let actualTypeName = typeName(for: variable.unwrappedTypeName, containingType: type) {
+                if let actualTypeName = typeName(for: variable.unwrappedTypeName, containingType: type, typealiases: typealiases) {
                     variable.type = unique[actualTypeName]
                 } else {
                     variable.type = unique[variable.unwrappedTypeName]
                 }
             }
-        }
+            // find actual methods parameters types and their argument labels
+            for method in type.methods {
+                let argumentLabels: [String]
+                if let labels = method.fullName.range(of: "(")
+                    .map({ method.fullName.substring(from: $0.upperBound) })?
+                    .trimmingCharacters(in: CharacterSet(charactersIn: ")"))
```

Comment:
There are no parameter types if `fullName`, so the only parentheses there are for the method parameters